sys/init.c

```diff
@@ -757,7 +776,6 @@ DokanCreateGlobalDiskDevice(__in PDRIVER_OBJECT DriverObject,
   return STATUS_SUCCESS;
 }
 
-KSTART_ROUTINE DokanRegisterUncProvider;
 VOID DokanRegisterUncProvider(__in PVOID pDcb) {
```

Comment:
* This can now directly be typed as `PDokanDCB Dcb` and we can remove https://github.com/dokan-dev/dokany/blob/6a412d7c21bc36db41a1d5a23f6e192e3ae4890c/sys/init.c#L778
* The `PsTerminateSystemThread` call can also be removed
* Same for the `DokanDeregisterUncProvider` function